### ISSUE TITLE
Support LUA 5.1

### ISSUE THREAD

**Issue by rajeshps** [aryajur/oauth2#1]
Hi, This seems specific to LUA 5.2+ how about supporting 5.1? any compatibility issues?

Thanks, Rajesh

**Comment by aryajur** [comment]
It should not be too complicated to do that. I think the module creation needs to change. Maybe something else. I will try to push in the change. If you have a pull request then do submit it.

Thanks.

**Comment by rajeshps** [comment]
Yes correct, it wasn't complicated yet I wanted to check or seek author's opinion on compatibility or known issues if any.
I have submitted pull request but not as a patch to original src instead its placed in separate versioned directory.
Please feel free to review and merge or apply as applicable.
Thanks!
